### PR TITLE
Update my image link #234

### PR DESCRIPTION
**Pull request by wermarter**

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -203,6 +203,7 @@
 
 - name: Ha Minh Chien
   github: wermarter
+  image: hmc.jpg
   image: chien.jpg
```
Comment (jigyasa-grover) [inline _data/students.yml]: Why the `image:` tag is there twice ?

**Review by jigyasa-grover — COMMENTED** [review]
* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Also provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.

File: _data/students.yml (hunk 1 above)
Comment (jigyasa-grover) [reply _data/students.yml]: @Wermarter

Comment (wermarter) [reply _data/students.yml]: @jig08 
I think it's because of the auto-merge

**Comment by wermarter** [comment]
Fixed on #288

**Pull request closed without merging**